.env.sample

```diff
@@ -1,22 +1,31 @@
-BUGSNAG_API=abcd1234efgh5678ijkl9101112mn
+# Salt used for encrypting the configuration.
 SALT=f1nd1ngn3m0
+# Default login credentials, use in all of the python tests.
 DEFAULT_LOGIN_USERNAME=default
-DEFAULT_LOGIN_PASSWORD=********
 DEFAULT_LOGIN_TOKEN=0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef
+# Credentials, used for login testing.
 VALID_LOGIN_USERNAME=valid
-VALID_LOGIN_PASSWORD=*********
 DEFAULT_LOGIN_TOKEN=0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef
 EXPIRED_LOGIN_USERNAME=expired
-EXPIRED_LOGIN_PASSWORD=********
 DEFAULT_LOGIN_TOKEN=0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef
+# Credentials used for meshnet testing. QA peer and Default accounts connect with each other via meshnet.
 QA_PEER_USERNAME=qa
 QA_PEER_PASSWORD=********
 DEFAULT_LOGIN_TOKEN=0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef
-CI_PERSONAL_TOKEN=********
+# Source of binaries of app dependencies.
 CI_API_V4_URL=http://example.com
+# Credentials for binaries of app dependencies.
 NVPN_LINUX_GL_DEPS_CREDS=user:password
+# Domains for analytics.
 EVENTS_PROD_DOMAIN=http://example.com
 EVENTS_STAGING_DOMAIN=http://example.com
 EVENTS_SUBDOMAIN=subdomain
+# App features, possible values are:
+# * telio
+# * drop
+# * moose - internal builds only
+# * internal - internal builds only
 FEATURES="telio drop"
```

Comment:
I get errors when compiling with FEATURES in quotation marks. Marius also wrote about that in onboarding resources. So maybe they should be removed from the example file?